styles.css

```diff
@@ -0,0 +1,91 @@
+h2 {
+  font-family: "Playfair Display SC", serif;
+  font-size: 42px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+
+  line-height: 5px;
+  color: white;
+}
+nav {
+  font-family: "Open Sans", sans-serif;
+  font-style: italic;
+  font-size: 16px;
+  line-height: 10px;
```

Comment:
Same note here about very small `line-height`. See the [class reading on `line-height`](https://mica-web.github.io/learn/css/typography#line-height).